packages/components/root/stories/root.stories.js

```diff
@@ -0,0 +1,125 @@
+import { storiesOf } from '@storybook/html';
+import '@tradeshift/elements';
+import '@tradeshift/elements.root';
+
+const slots = {
+	header: { slot: 'header', height: '60px', background: '#00AEFF' },
+	footer: { slot: 'footer', height: '60px', background: '#9AB2BC' },
+	'sidebar-left': {
+		slot: 'sidebar-left',
+		height: 'auto',
+		width: '320px',
+		background: '#50C610'
+	},
+	'sidebar-right': {
+		slot: 'sidebar-right',
+		height: 'auto',
+		width: '320px',
+		background: '#FDBE12'
+	},
+	'sidebar-inner-left': {
+		slot: 'sidebar-inner-left',
+		height: 'auto',
+		width: '320px',
+		background: '#A70262'
+	},
+	'sidebar-inner-right': {
+		slot: 'sidebar-inner-right',
+		height: 'auto',
+		width: '320px',
+		background: '#90129B'
+	}
+};
+storiesOf('ts-root', module)
+	.add('empty', () => {
+		const root = createRoot();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "root". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3AS1488)